### PR TITLE
DOC-2194: added fix documentation for TINY-10263 to the 6.7.2 Release notes

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-2194:

Changes:
* added fix documentation for TINY-10263 to the 6.7.2 Release notes

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] Changelog entry added

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

**Review by HAFRMO — APPROVED** [review]

File: modules/ROOT/pages/6.7.2-release-notes.adoc [hunk 1]
```diff
@@ -41,9 +41,15 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
...
 === The function `getModifierState` did not work on events passed through the editor as expected.
 // TINY-10263
+The exception error "Uncaught TypeError: Illegal invocation" was displayed in the console, when the editor called the 'getModifierState' function on a **keydown** event action.
 
-// CCFR
+This error was introduced, when the function was re-used in a previous update to {productname}, but, the updated logic for the origional function was not changed to consider the newly introduced context.
+
+As a consequence, any attempts to invoke this function resulted in the TypeError being displayed in the users developer console.
+
+{productname} 6.7.2 addresses this issue, by updating the 'getModifierState' function to now consider the updated context introduced in a previous release.
```
[the remark this answers is not in the document]
Comment (kemister85) [reply modules/ROOT/pages/6.7.2-release-notes.adoc:50]: @HAFRMO I agree, but im not sure if providing less info will be sufficient enough for a docs entry, as I dont believe telling the customer that we "copied over a function without concern of context" is a good idea either as mentioned in TINY-10263 docs outline? as this might give them the wrong impression?

File: modules/ROOT/pages/6.7.2-release-notes.adoc [hunk 2]
```diff
@@ -41,9 +41,15 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 === The function `getModifierState` did not work on events passed through the editor as expected.
 // TINY-10263
+The exception error "Uncaught TypeError: Illegal invocation" was displayed in the console, when the editor called the 'getModifierState' function on a **keydown** event action.
 
-// CCFR
+This error was introduced, when the function was re-used in a previous update to {productname}, but, the updated logic for the origional function was not changed to consider the newly introduced context.
```
Comment (spocke) [inline modules/ROOT/pages/6.7.2-release-notes.adoc:46]:
```suggestion
This error was introduced, when the function was re-used in a previous update to {productname}, but, the updated logic for the original function was not changed to consider the newly introduced context.
```

File: modules/ROOT/pages/6.7.2-release-notes.adoc [hunk 3]
```diff
@@ -41,9 +41,15 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 === The function `getModifierState` did not work on events passed through the editor as expected.
 // TINY-10263
+The exception error "Uncaught TypeError: Illegal invocation" was displayed in the console, when the editor called the 'getModifierState' function on a **keydown** event action.
```
Comment (spocke) [inline modules/ROOT/pages/6.7.2-release-notes.adoc:44]: It's the integrator or developer who calls the function and that causes an error. I don't think we even use that function that is why we missed it not being executed with the right context.

```suggestion
The exception error "Uncaught TypeError: Illegal invocation" was displayed in the console, when an integrator called the 'getModifierState' function on a **keydown** event action.
```

File: modules/ROOT/pages/6.7.2-release-notes.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (spocke) [reply modules/ROOT/pages/6.7.2-release-notes.adoc:50]: I think we just need to explain that an exception was thrown if the integrator called `getModifierState` on the event object that the editor provide though the `keydown` event handler call. How to explain that in simpler terms I'm not sure.

File: modules/ROOT/pages/6.7.2-release-notes.adoc (hunk 3 above)
Comment (vpyshnenko) [reply modules/ROOT/pages/6.7.2-release-notes.adoc:44]: The method `getModeierState` is called on an event object,  not an event action.

File: modules/ROOT/pages/6.7.2-release-notes.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (HAFRMO) [reply modules/ROOT/pages/6.7.2-release-notes.adoc:50]: I think Spocke's right on this one.

File: modules/ROOT/pages/6.7.2-release-notes.adoc [hunk 4]
```diff
@@ -41,9 +41,15 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 === The function `getModifierState` did not work on events passed through the editor as expected.
 // TINY-10263
+The exception error "Uncaught TypeError: Illegal invocation" was displayed in the console, when an integrator called the 'getModifierState' function on a **keydown** event object.
 
-// CCFR
+This error was introduced, when the function was re-used in a previous update to {productname}, but, the updated logic for the original function was not changed to consider the newly introduced context.
```
Comment (kemister85) [inline modules/ROOT/pages/6.7.2-release-notes.adoc:46]: @spocke suggest we remove this
```suggestion
```

File: modules/ROOT/pages/6.7.2-release-notes.adoc [hunk 5]
```diff
@@ -41,9 +41,15 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
...
+
+As a consequence, any attempts to invoke this function resulted in the TypeError being displayed in the users developer console.
```
Comment (kemister85) [inline modules/ROOT/pages/6.7.2-release-notes.adoc:48]:
```suggestion
As a consequence, any attempts to invoke this function resulted in the TypeError being displayed in the console.
```

File: modules/ROOT/pages/6.7.2-release-notes.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (kemister85) [reply modules/ROOT/pages/6.7.2-release-notes.adoc:50]:
```suggestion
{productname} 6.7.2 addresses this, now, when a integrator uses the `getModifierState` on the event object a keydown event no longer triggers an error.
```

File: modules/ROOT/pages/6.7.2-release-notes.adoc [hunk 6]
```diff
@@ -41,9 +41,15 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
...
+
+{productname} 6.7.2 addresses this issue, by updating the 'getModifierState' function to now consider the updated context introduced in a previous release.
 
+As a result, the TypeError is no longer displayed when the editor calls the 'getModifierState' function on an event during a **keydown** action.
```
Comment (kemister85) [inline modules/ROOT/pages/6.7.2-release-notes.adoc:52]:
```suggestion
```

**Review by tiny-ben-tran — APPROVED** [review]
